package.json

```diff
@@ -74,7 +83,19 @@
     "clean": "rimraf dist",
     "deploy": "gh-pages -d example/dist",
     "publish-demo": "npm run build-demo && npm run deploy",
-    "publish-npm-package": "npm run build && npm publish"
+    "publish-npm-package": "npm run build && npm publish",
+    "format": "prettier --config .prettierrc --parser typescript --write 'src/**/*.tsx'",
+    "lint": "eslint 'src/**/*.tsx' --ext .ts",
+    "lint:fix": "eslint 'src/**/*.tsx' --ext .ts --fix"
+  },
+  "simple-git-hooks": {
+    "pre-commit": "npx lint-staged"
```

Comment:
I would prefer that formatting, testing, etc is done in CI and not with git hooks. Mostly because it reduces dependencies and offsets formatting, testing and building to the integration suite.

Could you create a configuration for [GitHub Actions](https://github.com/features/actions)? I have [an example workflow file](https://github.com/jamesrweb/codewars-api-client/blob/feature/v3/.github/workflows/workflow.yml) here from another side project of mine for reference.

Reply:
Agreed, it's a good idea to configure CI. The only thing I'm concerned about is formatting files in the CI with extra commits, I'm not sure this is the best approach. I think a pre-commit git hook would work better for that, but if you don't like it, I can move this step too.

Reply:
I really believe CI is the place for this, I updated the code myself, could you test my changes?